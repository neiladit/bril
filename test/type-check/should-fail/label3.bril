#multiple labels present, one missing

main{

    v0: int = const 1;
    v1: int = const 3;
    jmp l1;
    l2:
    v4: int = const 4;
    print v4;
    jmp l3;
    l1:
    v3: int = add v1 v2;
    jmp l2;
    
}
