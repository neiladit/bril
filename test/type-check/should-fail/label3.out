line 8: jump to undefined label l3
jmp error!
